x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/influencers/influencers.tsx

```diff
@@ -14,16 +14,22 @@ import {
   isMultiMetricJobCreator,
   PopulationJobCreator,
   isPopulationJobCreator,
+  AdvancedJobCreator,
+  isAdvancedJobCreator,
 } from '../../../../../common/job_creator';
 import { Description } from './description';
 
 export const Influencers: FC = () => {
   const { jobCreator: jc, jobCreatorUpdate, jobCreatorUpdated } = useContext(JobCreatorContext);
-  if (isMultiMetricJobCreator(jc) === false && isPopulationJobCreator(jc) === false) {
+  if (
```

Comment:
As mentioned in the other component, returning `null` here before calling all hooks isn't valid. All hooks need to be called on every render.